### PR TITLE
Add org & storybook docs

### PR DESCRIPTION
**Pull request by necoline**
I added docs on the organization and domain creation as well as an overview of Storybook. 

I'm thinking that we can reuse that gif that @filipelima18 put in the blog as the main visual.

I wasn't sure how much of the instructions for adding a domain and configuring the addon I should repeat and what I should just reference. I opted to bring the githun content into the docs so all the content exists on the docs and they don't need to leave the page to learn what they need to do, but I'm curious to know if you think this strikes the right balance.

### CODE REVIEW

File: packages/projects-docs/pages/learn/integrations/storybook.mdx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+};
+```
+
+</details>
+
+<br />
+
+Make sure to provide the necessary values for [`apiToken`](https://codesandbox.io/t/permissions) and any additional dependencies or providers required for your specific setup.
+
+For now, this addon only support [Component Story Format (CSF)](Component Story Format (CSF)) stories format.
```
Comment (filipelima18) [inline packages/projects-docs/pages/learn/integrations/storybook.mdx:161]:
```suggestion
For now, this addon only supports the [Component Story Format (CSF)](https://storybook.js.org/docs/api/csf) stories format.
```

File: packages/projects-docs/pages/learn/integrations/storybook.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+### Additional Notes
+- Ensure that you have proper permissions and access rights to the CodeSandbox workspace specified in the configuration.
+- Verify the correctness of the dependencies and providers listed in the configuration to ensure the sandbox runs smoothly.
```
Comment (filipelima18) [inline packages/projects-docs/pages/learn/integrations/storybook.mdx:165]:
```suggestion
- Verify the correctness of the dependencies and providers listed in the configuration to ensure the Sandbox runs smoothly.
```

**Review by filipelima18 — CHANGES_REQUESTED** [review]
Made a few suggestions

File: packages/projects-docs/pages/learn/access/organizations.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+![edit org](../images/org-edit.png) 
+
+The organziation settings will always be discoverable through the workspace settings
+
+## Setting an email domain
+
+In the organizations settings, open the `Domain` tab and add your email domain. 
+The domain will need to be verified before being approved. This is an automated process and requires no action on your part.
```
[the remark this answers is not in the document]
Comment (necoline) [reply packages/projects-docs/pages/learn/access/organizations.mdx:42]: Ah thank you. I struggled with this sentence.

**Review by filipelima18 — APPROVED** [review]

**Pull request merged**